lib/controller.rb

```diff
@@ -59,6 +59,18 @@ class SlowFood < Sinatra::Base
     erb :menu
   end
 
+  enable :sessions
+
+  post '/menu' do
+    @dish = Dish.get(params[:dishid].to_i)
+    @dish.save
```

Comment:
You don't need this line here. You're not changing `@dish` so you don't need to save it.